src/components/Addresses.vue

```diff
@@ -49,7 +49,7 @@ export default {
   },
   data() {
     return {
-      selectedIndex: this.addresses.length - 1 || null,
+      selectedIndex: this.addresses.length - 1 >= 0 ? this.addresses.length - 1 : -1,
```

Comment:
The expression `this.addresses.length` does always fulfill the `>= 0` requirement, while the base value is `-1`, so the whole expression actually equals to this:
```suggestion
      selectedIndex: this.addresses.length - 1,
```

Reply:
Ashamed :no_mouth: